.github/workflows/build_slim_packages.yaml

```
@@ -194,6 +194,11 @@ jobs:
         HTTP_PORT=$(docker inspect --format='{{(index (index .NetworkSettings.Ports "18083/tcp") 0).HostPort}}' $CID)
         ./scripts/test/emqx-smoke-test.sh localhost $HTTP_PORT
         docker stop $CID
+    - name: test two nodes cluster with proto_dist=inet_tls in docker
+      run: |
+        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG
```

We can pick an old image, so to cover the compatibility checks too.
i.e. newer version node joins older version cluster (how rolling upgrade is done).
```suggestion
        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG $EMQX_IMAGE_OLD_VERSION_TAG
```

---

Done.